pkg/install/install.go

```diff
@@ -54,12 +56,43 @@ func Kourier() error {
 	}
 	fmt.Println("    Ingress patched...")
 
-	// TODO move svc yaml to kn-plugin-quickstart repo and update location
-	kourierIngress := exec.Command("kubectl", "apply", "-f", "https://gist.githubusercontent.com/psschwei/8321b367bb9e4281025b5b17e9cbb673/raw/e9efa21df77322a42de183b60c4e0933dbaae830/kourier-ingress.yaml")
-	if err := wait.PollImmediate(1*time.Second, 5*time.Second, func() (bool, error) {
-		return runCommand(kourierIngress) == nil, nil
-	}); err != nil {
-		return fmt.Errorf("wait: %w", err)
+	kourierConfig, err := ioutil.TempFile(os.TempDir(), "kourier-ingress-*.yaml")
+	if err != nil {
+		return fmt.Errorf("kourier service: %w", err)
+	}
+
+	defer os.Remove(kourierConfig.Name())
+
+	configRaw := `apiVersion: v1
+kind: Service
+metadata:
+  name: kourier-ingress
+  namespace: kourier-system
+  labels:
+    networking.knative.dev/ingress-provider: kourier
+spec:
+  type: NodePort
+  selector:
+    app: 3scale-kourier-gateway
+  ports:
+    - name: http2
+      nodePort: 31080
+      port: 80
+      targetPort: 8080`
+	config := []byte(configRaw)
+
+	if _, err := kourierConfig.Write(config); err != nil {
+		return fmt.Errorf("kourier service: %w", err)
+	}
+
+	configFile := kourierConfig.Name()
+	kourierIngress := exec.Command("kubectl", "apply", "-f", configFile)
```

Comment:
an alternative approach which avoids the temp file stuff and may be a bit cleaner is

~~~~golang
	kourierIngress := exec.Command("kubectl", "apply", "-f", "-")
        kourierIngress.Stdin = strings.NewReader(config)
        runCommand(kourierIngress)
~~~~

Reply:
ooh... much nicer